Docs/src/instancing.md

```diff
@@ -0,0 +1,373 @@
+ - ShaderInstancing.vert (GLSL)
+ - ShaderInstancing.cg (Cg, works with HLSL)
+
+## VTF (Software) {#InstancingTechniquesVTFSoftware}
+
+VTF stands for "Vertex Texture Fetch". 
+It is a Software Instancing technique. 
+Unlike ShaderBased, world matrices are passed to the vertex shader through a texture. 
+Such feature has only been supported since Vertex Shader 3.0 and is not supported on Radeon X1xxx cards and is quite slow on GeForce 6 & 7. 
+However it's very fast on any modern GPU (GeForce 8, 9, 200, 300, 400, 500, 600, 700; all Radeon HD series, Intel HD 3000 and above)
+
+The advantage of VTF over ShaderBased is that it supports a very high max number of instances per batch; even if it's skeletally animated.
+
+Take note that you will need to set a `texture_unit` (preferrably the first one, for compatibility) including the shadow caster besides the texture (eg. diffuse, specular, normal maps) so that Ogre gets where to put the vertex texture.
+
+See `material Examples/Instancing/VTF` for an example on how to write the vertex shader and setup the material. 
+Files:
+ - VTFInstancing.material
+ - VTFInstancing.vert (GLSL)
+ - VTFInstancing.cg (Cg, also works with HLSL)
+
+## HW VTF {#InstancingTechniquesHWVTF}
+
+This is the same technique as VTF; but implemented through hardware instancing. 
+It is probably one of the best and most flexible techniques.
+
+The vertex shader has to be slightly different from SW VTF version. 
+See `material Examples/Instancing/HW_VTF` for an example on how to write the vertex shader and setup the material. 
+Files:
+ - HW\_VTFInstancing.material
+ - HW\_VTFInstancing.vert (GLSL)
+ - HW\_VTFInstancing.cg (Cg, works with HLSL)
+
+### HW VTF LUT {#InstancingTechniquesHW}
+
+LUT is a special feature of HW VTF; which stands for <b>L</b>ook <b>U</b>p <b>T</b>able. 
+It has been particularly designed for drawing large animated crowds.
+
+The technique is a trick that works by animating a limited number of instances (i.e. 16 animations) storing them in a look up table in the VTF, 
+and then repeating these animations to all instances uniformly, giving the appearance that all instances are independently animated when seen in large crowds.
+
+See `material Examples/Instancing/HW_VTF_LUT`. 
+Files:
+ - Same as HW VTF (different macros defined)
+
+To enable the use of LUT, `SceneManager::createInstanceManager`'s flags must include the flag `IM_VTFBONEMATRIXLOOKUP` and specify HW VTF as technique.
+
+```cpp
+mSceneMgr->createInstanceManager("InstanceMgr","MyMesh.mesh",
+			ResourceGroupManager::AUTODETECT_RESOURCE_GROUP_NAME,
+			InstanceManager::HWInstancingVTF,
+			numInstancesPerBatch,IM_USEALL|IM_VTFBONEMATRIXLOOKUP );
+```
+
+## HW Basic {#InstancingTechniquesHWBasic}
+
+HW Basic is probably the fastest instancing technique[^7], but is surely more compatible than HW VTF.
+
+The world matrix data is passed to the vertex shader using three `TEXCOORD`s (*attribute* in GLSL jargon) instead of a vertex texture.
```

Comment:
```suggestion
The world matrix data is passed to the vertex shader using a `float3x4` `TEXCOORD` (*attribute* in GLSL jargon) instead of a vertex texture.
```